Netflix Clone/Services/APIClient.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  APIClient.swift
+//  Netflix Clone
+//
+//  Created by Polina Poznyak on 6.07.23.
+//
+
+import Foundation
+
+struct APIClient {
```

Comment:
I would strongly suggest to have APIClientProtocol :) 
And, it is not a mistake, but usually for APIClients we use classes, just because APIClient is usually used by a lot of consumers (ViewModels, Presenters)